pgscm/templates/farmer/macro.html

```diff
@@ -0,0 +1,42 @@
+{% import "bootstrap/wtf.html" as wtf%}
```

Comment:
this file is code-smell with the same file in agroup bp.

I have moved macro template file to base template, please remove this find for avoiding code duplication.